runtime/compiler/trj9/runtime/HookHelpers.cpp

```diff
@@ -260,10 +272,12 @@ void jitReleaseCodeCollectMetaData(J9JITConfig *jitConfig, J9VMThread *vmThread,
       cleanUpJitExceptionHandlerCache(vmThread,metaData);
       cleanUpJitArtifactSearchCache(vmThread,metaData);
 
-      // 3rd parameter to reclaimAssumptions determines whether we reclaim preprologue assumptions.
+      // Mark the runtime assumptions so thay can be deleted at the end of the GC cycle.
+      // The 3rd parameter to markAssumptionsAndDetach determines whether we mark preprologue assumptions.
```

Comment:
Is it the third or the fourth parameter?

Reply:
It's the 3rd. 1st = jitConfig, 2nd = metaData, 3rd = (NULL == faintCacheBlock)